packages/ckb-sdk-core/src/generateRawTransaction.ts

```diff
@@ -27,10 +31,24 @@ const generateRawTransaction = async ({
     throw new HexStringShouldStartWith0x(capacity)
   }
 
-  const targetCapacity = typeof capacity !== 'bigint' ? BigInt(capacity) : capacity
-  const realFee = typeof fee !== 'bigint' ? BigInt(fee) : fee
+  if (typeof capacityThreshold === 'string' && !capacityThreshold.startsWith('0x')) {
```

Comment:
This logic seems to occur in multiple places, what about adding a utility function that takes `bigint | string` and outputs `bigint`, and we can encapsulate this logic in a unified place?

Reply:
`number` is removed in https://github.com/nervosnetwork/ckb-sdk-js/pull/382/commits/3af1bfc9d0f2fdde4d2ff8a0a53bdf2c3d885a34